host-configs/rzgenie-toss_3_x86_64_ib-clang@4.0.0.cmake

```diff
@@ -99,6 +99,8 @@ set(MPIEXEC_NUMPROC_FLAG "-n" CACHE PATH "")
 
 set(ENABLE_GTEST_DEATH_TESTS ON CACHE BOOL "")
 
+set(BLT_EXE_LINKER_FLAGS "-Wl,-rpath,/usr/tce/packages/clang/clang-6.0.0/lib" CACHE PATH "Adds a missing rpath for libraries associated with the fortran compiler")
```

Comment:
@gzagaris -- I added the host-configs that you generated.

This should fix our CI tests on toss3.